### PR TITLE
Merge XLSX and PDF reports into a single reports tab

### PR DESCRIPTION
**Pull request by saiprasanth1303**
Changes proposed in this pull request:
- Merged the XLSX and PDF reports into a single reports tab

@deepfence/engineering

### CODE REVIEW

File: deepfence_ui/app/scripts/actions/app-actions.js [hunk 1]
```diff
@@ -1465,51 +1466,59 @@ export function getTopVulnerableContainerAndHostsAction(params) {
...
+    ActionTypes.REPORT_EMAIL_SCHEDULE_REQUEST,
+    ActionTypes.REPORT_EMAIL_SCHEDULE_SUCCESS,
+    ActionTypes.REPORT_EMAIL_SCHEDULE_FAILURE,
   ];
-  return genericThunkAction(actionTypes, getPdfDownloadStatus, params);
+  return genericThunkAction(actionTypes, reportScheduleEmail, params);
 }
 
-export function downloadPdfReportAction(params) {
+export function xlsxReportDownloadAction(params) {
```
Comment (manV) [inline deepfence_ui/app/scripts/actions/app-actions.js:1505]: is this still required? I mean the xlsx methods?

File: deepfence_ui/app/scripts/constants/action-types.js [hunk 2]
```diff
@@ -201,22 +201,25 @@ const ACTION_TYPES = [
...
+  'DOWNLOAD_REPORT_FAILURE',
+  'REPORT_EMAIL_SCHEDULE_REQUEST',
+  'REPORT_EMAIL_SCHEDULE_SUCCESS',
+  'REPORT_EMAIL_SCHEDULE_FAILURE',
   'XLSX_REPORT_DOWNLOAD_REQUEST',
   'XLSX_REPORT_DOWNLOAD_SUCCESS',
   'XLSX_REPORT_DOWNLOAD_FAILURE',
   'XLSX_EMAIL_SCHEDULE_REQUEST',
   'XLSX_EMAIL_SCHEDULE_SUCCESS',
   'XLSX_EMAIL_SCHEDULE_FAILURE',
```
Comment (manV) [inline deepfence_ui/app/scripts/constants/action-types.js:221]: Also these?

File: deepfence_ui/app/scripts/actions/app-actions.js (hunk 1 above)
Comment (manV) [reply deepfence_ui/app/scripts/actions/app-actions.js:1505]: @saiprasanth1303

Comment (saiprasanth1303) [reply deepfence_ui/app/scripts/actions/app-actions.js:1505]: We are still using this function when downloading CVE reports directly from the vulnerabilities page

File: deepfence_ui/app/scripts/constants/action-types.js (hunk 2 above)
Comment (saiprasanth1303) [reply deepfence_ui/app/scripts/constants/action-types.js:221]: Similarly, we are using these actions to download XLSX reports on the vulnerabilities page.

**Review by saurabh2253 — APPROVED** [review]

**Pull request merged**